### PR TITLE
Refactor MTE-1360 [v118] Marketing Screenshot Tests

### PR DESCRIPTION
**Pull request by clarmso**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1360)

## :bulb: Description
Let me extract the (unused) `MarketingUITests` to the existing L10n Snapshot tests.

When I ran `./l10n-screenshots.sh` locally, the new test `testTakeMarketingScreenshots()` was run.
```
[16:27:25]: ▸ Test Suite L10nSnapshotTests.xctest started
[16:27:25]: ▸ L10nSuite1SnapshotTests
[16:27:55]: ▸     ✓ test3ReloadButtonContextMenu (29.547 seconds)
[16:28:21]: ▸     ✓ testBookmarksTableContextMenu (26.448 seconds)
[16:28:59]: ▸     ✓ testETPperSite (37.183 seconds)
[16:29:25]: ▸     ✓ testHistoryTableContextMenu (26.040 seconds)
[16:29:54]: ▸     ✓ testIntro (29.635 seconds)
[16:30:11]: ▸     ✓ testMenuOnTopSites (16.867 seconds)
[16:30:23]: ▸     ✓ testPrivateBrowsingTabsEmptyState (12.133 seconds)
[16:31:34]: ▸     ✓ testSettings (70.901 seconds)
[16:31:57]: ▸     ✓ testSettingsETP (22.353 seconds)
[16:32:46]: ▸     ✓ testTakeMarketingScreenshots (49.476 seconds)
[16:32:58]: ▸     ✓ testTopSitesMenu (11.950 seconds)
[16:33:11]: ▸     ✓ testWebViewAuthenticationDialog (12.878 seconds)
[16:33:17]: ▸ L10nSuite2SnapshotTests
[16:33:36]: ▸     ✓ testFxASignInPage (19.223 seconds)
[16:34:17]: ▸     ✓ testLoginDetails (41.305 seconds)
[16:34:34]: ▸     ✓ testLongPressOnTextOptions (17.228 seconds)
[16:34:56]: ▸     ✓ testMenuOnWebPage (21.318 seconds)
[16:35:13]: ▸     ✓ testPageMenuOnWebPage (16.712 seconds)
[16:35:34]: ▸     ✓ testPanelsEmptyState (21.245 seconds)
[16:35:49]: ▸     ✓ testURLBar (15.384 seconds)
[16:35:55]: ▸     ✓ testURLBarContextMenu (5.822 seconds)
[16:35:55]: ▸    Executed 21 tests, with 1 test skipped and 0 failures (0 unexpected) in 509.460 (509.488) seconds
```

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: L10nSnapshotTests/L10nSuite1SnapshotTests.swift [hunk 1]
```diff
@@ -245,4 +245,30 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
         navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
         snapshot("PrivateBrowsingTabsEmptyState-01")
     }
+
+    func testTakeMarketingScreenshots() {
+        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.settingsMenuButton], timeout: 10)
```
Comment (isabelrios) [inline L10nSnapshotTests/L10nSuite1SnapshotTests.swift:250]: Is this tests proving new screenshots that we do not have yet?

**Comment by isabelrios** [comment]
The MarketingUITest scheme and test still appears for me... I think you have to remove the file and its reference so that the change is reflected in the Client.xcodproj file and the scheme is completely removed
Screenshot 2023-08-11 at 10 21 20

File: L10nSnapshotTests/L10nSuite1SnapshotTests.swift (hunk 1 above)
Comment (clarmso) [reply L10nSnapshotTests/L10nSuite1SnapshotTests.swift:250]: Not new:
* 00TopSites (same with Homescreen-first-visit)
* 01SearchResults is similar to URLBar-02, but not exactly the same

New
* 02-TabTray
* 03-SyncedTabs

**Comment by clarmso** [comment]
> The MarketingUITest scheme and test still appears for me... I think you have to remove the file and its reference so that the change is reflected in the Client.xcodproj file and the scheme is completely removed 

I've removed the `MarketingUITests.xcscheme` file but the scheme is still referenced in XCode. 😞 `project.pbxproj` still contains references to this scheme. How do we remove those references without editing the file manually?

**Review by isabelrios — APPROVED** [review]

**Pull request merged**